doc/security-warnings.md

```diff
@@ -0,0 +1,37 @@
+Security Warnings
+====================
+
+Security Audit
+--------------
+
+Zcash has not yet been subjected to a formal third-party security review. This
+section will be updated with links to security audit reports in the future.
+
+x86-64 Linux Only
+-----------------------
+
+There are [known bugs](https://github.com/scipr-lab/libsnark/issues/26) which
+make proving keys generated on 64-bit systems unusable on 32-bit and big-endian
+systems. It's unclear if a warning will be issued in this case, or if the
+proving system will be silently compromised.
+
+Side-Channel Attacks
+--------------------
+
+This implementation of Zcash is not resistant to side-channel attacks. You
+should assume other unprivileged users running on the same hardware as your
```

Comment:
Also, they should assume anyone with physical access or close proximity to the device would be capable to do these things.